src/features/broker-protection/actions/fill-form.js

```diff
@@ -17,12 +22,26 @@ export function fillForm (action, userData) {
         // let inputElem = form.elements[element.selector]
         // find the correct userData to put in the form
         if (inputElem) {
-            // @ts-expect-error - double check if this is strict enough
-            // todo: determine if this requires any events to be dispatched also
-            setValueForInput(inputElem, userData[element.type])
+            if (element.type === '$file_id$') {
```

Comment:
Why does the id look like this (vs. 'file')?

Reply:
because all other 'types' here represent a key in the data object - but this is the first instance we have where the 'type' indicates a none-data-lookup. so, the `$`'s are there to make the difference clear - not sure if this is a good/bad idea yet, I propose we merge & work with it and the re-address if we come across a better solution :)